src/app/api/v1/endpoints/translations.py

```
@@ -0,0 +1,37 @@
+    return TranslationService(settings, gemini_client)
+
+
+@router.post(
+    "/translate",
+    response_model=TranslationResult,
+    summary="Translate verses using Dynamic RAG",
+    dependencies=[Depends(require_api_key)],
+)
+async def translate_verses(
+    request: TranslateRequest,
+    service: TranslationService = Depends(get_translation_service),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)